easy_phi/static/js/main.js

```diff
@@ -159,21 +145,17 @@ var ep = window['ep'] || {
         // create containers for modules by number of slots in this platform
         var module_list_container = $('#modules_list');
         module_list_container.empty();
-        // ep.slots + 1 because of Broadcast module in slot 0
-        for (var i=0; i<ep.slots+1; i++)
-            ep._add_module(module_list_container, i, ep._empty_slot_str);
 
         // get list of modules and update created containers
         $.get(ep.base_url+"/api/v1/modules_list?format=json", function(modules){
             modules.forEach(function(module_name, slot_id) {
-                if (slot_id > ep.slots) {
```

Comment:
... as mentioned above, now containers for all slots/added dynamically, just as for extra modules.